app/models/contact-list.js

```diff
@@ -0,0 +1,7 @@
+import Model, { attr, hasMany } from '@ember-data/model';
+
+export default class ContactListModel extends Model {
+  @attr('string') name;
+
+  @hasMany('contact-item') contactItems;
```

Comment:
Relaties naar telefoon/e-mail ontbreken nog. Zie ook https://github.com/kanselarij-vlaanderen/app-persberichten/pull/2 in de backend.